lib/emis/military_information_configuration.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'common/client/configuration/soap'
+
+module EMIS
+  class MilitaryInformationConfiguration < Configuration
+    URL = Settings.emis.military_information_url
```

Comment:
I think they're somehow required by the base class, but I'll check it out to make sure